jstreegrid.js

```diff
@@ -1169,7 +1271,7 @@
 						} else if (gridCellNext && gridCellNext.length > 0) {
 							last.insertBefore(gridCellNext);
 						} else {
-							last.appendTo(dataCell);
+							last.appendTo(column);
```

Comment:
Same:  if `column` is just the equivalent of `dataCell` but using the detached, then I understand this.